app/models/core/MtdbsaModel.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.core
+
+import play.api.libs.json.{Format, Json}
+
+
+sealed trait MtdbsaResponse
+
+case class MtdbsaModel(mtdbsa: Option[String]) extends MtdbsaResponse
+
+case class MtdbsaErrorModel(status: Int, reason: String) extends MtdbsaResponse
+
+object MtdbsaModel {
+  implicit val format: Format[MtdbsaModel] = Json.format[MtdbsaModel]
+}
+
+object MtdbsaErrorModel {
+  implicit val format: Format[MtdbsaErrorModel] = Json.format[MtdbsaErrorModel]
```

Comment:
Can you remove this as it's not being used

Reply:
done